src/mailadm/conn.py

```diff
@@ -265,10 +265,20 @@ def get_user_by_addr(self, addr):
 
     def get_expired_users(self, sysdate):
         q = UserInfo._select_user_columns + "WHERE (date + ttl) < ?"
-        users = []
+        overdue_users = []
+        expired_users = []
         for args in self._sqlconn.execute(q, (sysdate,)).fetchall():
-            users.append(UserInfo(*args))
-        return users
+            overdue_users.append(UserInfo(*args))
+        for user in overdue_users:
+            # hard expire users who were supposed to live less than a month
```

Comment:
I wanted to have it as 27 days because 4 weeks is something someone might add as a soft-expire thing - this is clearly for short-lived accounts you would want to expire exactly on time.